models/demos/t3000/falcon40b/tests/test_perf_falcon.py

```diff
@@ -337,20 +387,19 @@ def run_test_FalconCausalLM_end_to_end(
         "decode_batch32",
     ],
 )
-@pytest.mark.parametrize(
-    "num_layers",
-    (
-        1,
-        60,
-    ),
-    ids=["layers_1", "layers_60"],
-)
 @pytest.mark.parametrize(
     "model_version",
     ("tiiuae/falcon-40b-instruct",),
     ids=["falcon_40b"],
 )
-@pytest.mark.parametrize("model_config_str", ("BFLOAT8_B-SHARDED", "BFLOAT8_B-DRAM", "BFLOAT16-DRAM"))
+@pytest.mark.parametrize(
+    "model_config_str",
+    (
+        "BFLOAT8_B-SHARDED",
+        "BFLOAT8_B-DRAM",
+        # "BFLOAT16-DRAM"
```

Comment:
Should BFLOAT16-DRAM be kept here since it's used for prefill in the demo?

Reply:
I enabled `BFLOAT16-DRAM` and added the perf targets for it.

I also changed the demo to use `BFLOAT8_B-DRAM` config for prefill since this is our main supported version atm. PCC is >0.99 since we changed the residual path to be fp16 in that version. I also ran the demo with some example inputs and output looks good.